client/components/TestReview/index.jsx

```diff
@@ -0,0 +1,270 @@
+import { convertDateToString } from '../../utils/formatter';
+
+const Ul = styled.ul`
+    li {
+        list-style-type: disc;
+        margin-left: 20px;
+    }
+`;
+
+const FilterButtonContainer = styled.div`
+    padding: 0 0.75rem;
+    border: 1px solid #d2d5d9;
+    border-radius: 3px;
+    margin-bottom: 40px;
+`;
+
+const TestReview = () => {
+    const { testPlanVersionId } = useParams();
+
+    const { loading, data, error } = useQuery(TEST_REVIEW_PAGE_QUERY, {
+        variables: { testPlanVersionId },
+        fetchPolicy: 'cache-and-network'
+    });
+
+    const [activeFilter, setActiveFilter] = useState('All ATs');
+
+    if (loading) {
+        return (
+            <PageStatus
+                title="Loading - Tests for Test Plan Version | ARIA-AT"
+                heading="Tests for Test Plan Version"
+            />
+        );
+    }
+
+    if (error || !data?.testPlanVersion) {
+        const errorMessage =
+            error?.message ??
+            `Failed to find a test plan version with ID ${testPlanVersionId}`;
+        return (
+            <PageStatus
+                title="Tests for Test Plan Version | ARIA-AT"
+                heading="Tests for Test Plan Version"
+                message={errorMessage}
+                isError
+            />
+        );
+    }
+
+    const testPlanVersion = data.testPlanVersion;
+    const testCount = testPlanVersion.tests.length;
+    const atNames = unique(
+        testPlanVersion.tests.flatMap(test => test.ats.map(at => at.name))
+    );
+
+    let filteredTests;
+    if (activeFilter === 'All ATs') {
+        filteredTests = testPlanVersion.tests;
+    } else {
+        filteredTests = testPlanVersion.tests.filter(test =>
+            test.ats.find(at => at.name === activeFilter)
+        );
+    }
+
+    const filterOptions = Object.fromEntries(
+        ['All ATs', ...atNames].map(key => {
+            let count;
+            if (key === 'All ATs') {
+                count = testPlanVersion.tests.length;
+            } else {
+                count = testPlanVersion.tests.filter(test =>
+                    test.ats.find(at => at.name === key)
+                ).length;
+            }
+            return [key, `${key} (${count})`];
+        })
+    );
+
+    return (
+        <Container id="main" as="main" tabIndex="-1">
+            <Helmet>
+                <title>{`Tests for ${testPlanVersion.title} ${testPlanVersion.versionString} | ARIA-AT`}</title>
+            </Helmet>
+            <h1>
+                {`Tests for ${testPlanVersion.title} ` +
+                    `${testPlanVersion.versionString}` +
+                    `${testPlanVersion.deprecatedAt ? ' (Deprecated)' : ''}`}
+            </h1>
+            <h2>Introduction</h2>
+            <p>
+                {`This page contains the full content of all ${testCount} ` +
+                    `tests the ARIA-AT team has authored for ` +
+                    `${testPlanVersion.title} as of version ` +
```

Comment:
```suggestion
                    `${testPlanVersion.title}, version ` +
```

This read to me like newly added tests, in some future `VYY.MM.DD` would and could also show up for the pattern, on the 'current' version's page.

Reply:
Sure this is fine!